web/src/hooks/useLocationQuery.ts

```diff
@@ -0,0 +1,16 @@
+import { useLocation } from 'react-router';
```

Comment:
The console is deprecating the react-router v5 package, so we are swapping to the compat package (which basically uses v6 under the hood) provided by the console. There is no API change between v5 and v6 for the `useLocation` hook so only the package import is needed to be updated here.

```suggestion
import { useLocation } from 'react-router-dom-v5-compat';
```